chart/values.yaml

```diff
@@ -0,0 +1,95 @@
+# Default values for ssl-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+serviceMonitor:
+  enabled: true
+  # Automatically label with "release: {{ .Release.Name }}"
+  # This label is used in dnation k8s monitoring stack
+  releaseLabel: true
+  extraLabels: {}
+  # Metrics scrape interval
+  scrapeInterval: 15s
+  # Metrics scrape timeout
+  scrapeTimeout: 14s
+
+image:
+  repository: ribbybibby/ssl-exporter
```

Comment:
Please do not use latest version of image here, when in `Chart.yaml` you specified the specific version

Reply:
When tag is not defined, then appVersion will be used as tag

Reply:
Thats true, I did not notice that `tag` field under it.

Reply:
Ok, so now it is as should be